### ISSUE TITLE
Error running koocli in Python 3: No module named 'utils'

### ISSUE THREAD

**Issue by wesinator** [Koodous/python-sdk#12]
```sh
pip3 install --user koodous
koocli 
```

```python-traceback
Traceback (most recent call last):
  File "/home/ubuntu/.local/bin/koocli", line 7, in <module>
    from koodous.cli import cli
  File "/home/ubuntu/.local/lib/python3.6/site-packages/koodous/cli.py", line 28, in <module>
    from utils import pygmentize_json, is_apk, sha256 as file_hash
ModuleNotFoundError: No module named 'utils'
```

**Comment by Nekmo** [comment]
Confirmed.
```
$ koocli 
Traceback (most recent call last):
  File "/home/nekmo/.virtualenvs/tmp-37938925f1755a3/bin/koocli", line 7, in <module>
    from koodous.cli import cli
  File "/home/nekmo/.virtualenvs/tmp-37938925f1755a3/lib/python3.6/site-packages/koodous/cli.py", line 28, in <module>
    from utils import pygmentize_json, is_apk, sha256 as file_hash
ModuleNotFoundError: No module named 'utils'
```

**Comment by wesinator** [comment]
closed in 679485e
